### PR TITLE
Add docs for deploy api to microgateway via APIM and apictl, and fix links related to mgw

### PR DESCRIPTION
**Pull request by SuKSW**
## Purpose
- Add docs for deploy api to microgateway via APIM and apictl
- Fix a few links related to mgw in mkdocs.yaml
- this is the rebased PR of https://github.com/wso2/docs-apim/pull/3386

### CODE REVIEW

File: en/docs/deploy-and-publish/deploy-on-gateway/api-microgateway/deploy-api/deploy-api-to-microgateway.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+``` bash
+apictl mg deploy api -f <path_to_the_API_project_just_created>/petstore -e dev -k
+```
+
+That's it!
+
+More apictl commands related to microgateway can be found at [Managing Microgateway with apictl]({{base_path}}/install-and-setup/setup/api-controller/managing-microgateways/managing-microgateways-with-ctl)
+
+## Invoke the API
+{! ./includes/obtain-jwt.md !}
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/api-microgateway/deploy-api/deploy-api-to-microgateway.md:145]: Was his tested locally?

File: en/docs/deploy-and-publish/deploy-on-gateway/api-microgateway/deploy-api/deploy-api-to-microgateway.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+There are two ways to add an API to the Microgateway.
+
+1. [Via API Manager](#via-api-manager)
+
+    By enabling the Control Plane Event Hub in the Microgateway configuration file, the APIs deployed in API Manager will automatically be available in Microgateway. 
+
+2. [Via the CLI tool (apictl)](#via-the-cli-tool-apictl)
+
+    If you are not using WSO2 API Manager yet, or only want to tryout microgateway, simply use the CLI tool [apictl](https://github.com/wso2/product-apim-tooling/releases) to host your API.
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/api-microgateway/deploy-api/deploy-api-to-microgateway.md:11]:
```suggestion
    If you are not using WSO2 API Manager yet, or only want to try out Microgateway, simply use the CLI tool [apictl](https://github.com/wso2/product-apim-tooling/releases) to host your API.
```
Shouldn't it be Choreo Connect and not Microgateway?

File: en/docs/deploy-and-publish/deploy-on-gateway/api-microgateway/deploy-api/deploy-api-to-microgateway.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+## Via API Manager
+
+!!! info
+    **Before you begin**
+
+    - Make sure you have installed docker and docker-compose on your machine.
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/api-microgateway/deploy-api/deploy-api-to-microgateway.md:18]:
```suggestion
    - Make sure you have installed Docker and Docker Compose on your machine.
```

File: en/docs/deploy-and-publish/deploy-on-gateway/api-microgateway/deploy-api/deploy-api-to-microgateway.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+
+    - Download the latest [Microgateway 4.0.0 release](https://github.com/wso2/product-microgateway/releases)and extract it to a folder of your choice. The extracted folder will be referred to as `MG_HOME` here onwards.
+
+    - This guide assumes that you have already started the WSO2 API Manager instance. If not, download the latest [release](https://github.com/wso2/product-apim/releases)and follow the steps [here](https://github.com/wso2/product-apim#installation--running)
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/api-microgateway/deploy-api/deploy-api-to-microgateway.md:22]: This is ok currently but this needs to be updated once the GA release goes out.

File: en/docs/deploy-and-publish/deploy-on-gateway/api-microgateway/deploy-api/deploy-api-to-microgateway.md (hunk 1 above)
Comment (SuKSW) [reply en/docs/deploy-and-publish/deploy-on-gateway/api-microgateway/deploy-api/deploy-api-to-microgateway.md:145]: Yes, both the cases were tested locally.

File: en/docs/deploy-and-publish/deploy-on-gateway/api-microgateway/deploy-api/deploy-api-to-microgateway.md (hunk 4 above)
Comment (SuKSW) [reply en/docs/deploy-and-publish/deploy-on-gateway/api-microgateway/deploy-api/deploy-api-to-microgateway.md:22]: Will do once the GA is released

File: en/docs/deploy-and-publish/deploy-on-gateway/api-microgateway/deploy-api/deploy-api-to-microgateway.md (hunk 2 above)
Comment (Rajith90) [reply en/docs/deploy-and-publish/deploy-on-gateway/api-microgateway/deploy-api/deploy-api-to-microgateway.md:11]: We thought of renaming all the places in one go

**Review by Rajith90 — APPROVED** [review]

**Comment by SuKSW** [comment]
Closing this PR in order to rebase and add the change suggested

**Pull request closed without merging**